libs/expo/betterangels/src/lib/ui-components/DocumentModal.tsx

```diff
@@ -111,6 +111,11 @@ export default function DocumentModal(props: IDocumentModalProps) {
       Icon: ViewIcon,
       route: `/file/${document?.id}`,
     },
+    {
+      title: `Edit filename`,
+      Icon: ViewIcon,
```

Comment:
[waiting](https://betterangels-group.slack.com/archives/C05ECN9FLGL/p1749601083001939?thread_ts=1749601027.500239&cid=C05ECN9FLGL) for direction from product